jbot/src/main/java/me/ramswaroop/jbot/core/slack/models/Event.java

```diff
@@ -21,6 +21,9 @@
 
     private int id;
     private String type;
+    private String subtype;
```

Comment:
@ramswaroop subtype will be in response of specific message events only, please see Message subtypes section in this [link](https://api.slack.com/events/message)